logstreams/src/test/java/io/zeebe/logstreams/state/StateSnapshotControllerTest.java

```diff
@@ -54,7 +51,7 @@ public void shouldThrowExceptionOnTakeSnapshotIfClosed() {
     // then
     assertThat(snapshotController.isDbOpened()).isFalse();
     assertThatThrownBy(() -> snapshotController.takeSnapshot(1))
-        .isInstanceOf(IllegalStateException.class);
+        .isInstanceOf(NullPointerException.class);
```

Comment:
:eyes:

Reply:
I know hehe, it's mostly cause I changed it to `Objects.requireNonNull`, which throws an NPE